src/pages/api/auth/refresh-token.ts

```diff
@@ -23,7 +23,7 @@ async function refreshToken(req: NextApiRequest, res: NextApiResponse) {
 
   try {
     const accessToken = oauth2ClientCredentials.createToken(
-      JSON.parse(req.cookies['auth-token']) as Token
```

Comment:
See comment to [logout-callback.ts](https://github.com/serlo/frontend/pull/1779/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.local&file-filters%5B%5D=.lock&file-filters%5B%5D=.md&file-filters%5B%5D=.production&file-filters%5B%5D=.sh&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&file-filters%5B%5D=.yaml&file-filters%5B%5D=.yml&show-deleted-files=true&show-viewed-files=true#r913552635)